src/VariableQueryEditor.tsx

```diff
@@ -5,5 +5,5 @@ import { DataSource } from 'datasource';
 import { QueryEditor } from 'QueryEditor';
 
 export function VariableQueryCodeEditor(props: QueryEditorProps<DataSource, RedshiftQuery, RedshiftDataSourceOptions>) {
-  return <QueryEditor {...props}></QueryEditor>;
+  return <QueryEditor {...props} renderRunQueryButtons={false}></QueryEditor>;
```

Comment:
Added a prop `renderRunQueryButtons` to let the VariableQueryEditor always hide the Run and Stop buttons. The bottom of the variable editor page already has a button to run the query.

The `onChange` function passed to our VariableQueryEditor *also triggers the query to be run* in addition to updating the value of the query. This functionality differs from the regular query editor where the `onRunQuery` function is used to trigger the running of the query. The `onRunQuery` function that is passed to the VariableQueryEditor is a no-op so the Run button essentially isn't doing anything.